src/ciflow-bot.ts

```diff
@@ -5,26 +5,48 @@ import {CachedIssueTracker} from './utils';
 const ciflowCommentStart = '<!-- ciflow-comment-start -->';
 const ciflowCommentEnd = '<!-- ciflow-comment-end -->';
 
-export function parseCIFlowIssue(rawText: string): object {
+interface IUserConfig {
+  githubHandler: string;
+  optOut: boolean;
+  defaultLabels?: string[];
+}
+
+// parseCIFlowIssue parses the issue body for default labels and opt-out users
+export function parseCIFlowIssue(rawText: string): Map<string, IUserConfig> {
   const rows = rawText.replace('\r', '').split('\n');
-  const retval = {};
+  const m: Map<string, IUserConfig> = new Map();
   // eslint-disable-next-line github/array-foreach
   rows.forEach((row: string) => {
     const elements = row.trim().split(' ');
     if (
       elements.length < 1 ||
       elements[0].length < 1 ||
-      !elements[0].startsWith('@')
+      !(elements[0].startsWith('@') || elements[0].startsWith('-@'))
```

Comment:
Can we make it into const string rather than repeating the same literal on lines 24 and 30?
Also, perhaps it can be changed to a regex to allow `-    @username`
```suggestion
      !(elements[0].startsWith('@') || elements[0].metches(new Regex('^-\\s*@')))
```